### PR TITLE
refactor: Updates old role verbiage in manage area and docs

### PR DESCRIPTION
**Pull request by clh7090**
image
image
Changed 'admin' to director in README and in the Users & Staff page

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -45,8 +45,8 @@ en:
           If many people have already RSVP'd, consider sending a message out to this bus list asking for interest as well.
         notes: Notes are shared with applicants. Supports Markdown and HTML.
       user:
-        role: Limited access prevents the admin from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Event tracking limits to only event tracking.
-        is_active: Deactivating a user will prevent them from logging in. Their access will be immediately revoked from the admin and application pages.
+        role: Limited access prevents the director from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Event tracking limits to only event tracking.
```
Comment (cbaudouinjr) [inline config/locales/en.yml:48]: Sorry this was a little confusing, role should read: "Organizers are prevented from adding, modifying, or deleting any records; modifications through the check-in process are allowed. Volunteers only have access to the check-in process."

**Review by cbaudouinjr — CHANGES_REQUESTED** [review]

**Comment by cbaudouinjr** [comment]
Also remember to commit using the semantic-release format! For this it'd be "refactor: Updates old role verbiage in manage area and docs"

**Review by peterkos — CHANGES_REQUESTED** [review]
Great start!

One thing we could do is have each role's description only show up when that role is selected in the dropdown, however that'd take some HAML `if` statements and messing around with SimpleForm if you're comfortable. 

Otherwise, we could try what I suggested in the comment with a mockup. Thoughts? @cbaudouinjr @clh7090

**Comment by peterkos** [comment]
Note: change base branch before merge!

**Comment by peterkos** [comment]
Moving TBA

**Pull request closed without merging**